### PR TITLE
Use HRP hashes for transactions

### PR DESCRIPTION
**Pull request by krzlabrdx**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,15 +3,27 @@ Release Date: unreleased
 
 ## RCNet v2 to RCNet v3 Migration Guide
 
-_WORK IN PROGRESS_
+Use Bech32m-encoded transaction hashes in `/transaction/committed-details` and `/transaction/status` endpoints.
+
+TODO TODO TODO TODO
```
Comment (PawelPawelec-RDX) [inline CHANGELOG.md:8]: could we fill it or elaborate what's left to do here?

**Review by PawelPawelec-RDX — APPROVED** [review]

**Comment by PawelPawelec-RDX** [comment]
i'm missing upgrade of deployment.ci file with pointer to proper node version.

File: src/RadixDlt.CoreApiSdk/core-api-spec-copy.yaml [hunk 2]
```diff
@@ -73,7 +73,7 @@ info:
     [Gateway API](https://docs-babylon.radixdlt.com/).
 
     ## Integration and forward compatibility guarantees
-  
+
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.CoreApiSdk/core-api-spec-copy.yaml:76]: i'm missing upgrade of deployment ci.env file with pointer to proper node version.

File: CHANGELOG.md (hunk 1 above)
Comment (krzlabrdx) [reply CHANGELOG.md]: Updated.

File: src/RadixDlt.CoreApiSdk/core-api-spec-copy.yaml (hunk 2 above)
Comment (krzlabrdx) [reply src/RadixDlt.CoreApiSdk/core-api-spec-copy.yaml:76]: Updated ci.env

**Review by PawelPawelec-RDX — APPROVED** [review]

**Pull request merged**